specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/examples/LabelHistory_List.json

```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "containerAppName": "testcontainerApp0",
+    "api-version": "2024-10-02-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/containerApps/testcontainerApp0/labelHistory/prod",
+            "name": "prod",
+            "type": "Microsoft.App/containerApps/labelHistory",
+            "properties": {
+              "label": "prod",
+              "revision": "testcontainerApp0-pjxhsye",
+              "start": "2024-09-24T21:20:22+00:00",
+              "stop": "2024-09-24T21:24:22+00:00"
```

Comment:
Thought - Will the date time property have time zone offset? Do you have to keep it UTC always?

Reply:
In the code we are keeping it UTC right so it is uniform
Time zone is a good idea for future improvement though